receiver/oracledbreceiver/documentation.md

```diff
@@ -0,0 +1,51 @@
+| **oracledb.session.execute_count** | Total number of calls (user and recursive) that executed SQL statements | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.hard_parses** | Number of hard parses | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.logical_reads** | Number of logical reads | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.parse_count_total** | Total number of parse calls (hard, soft, and describe). A soft parse is a check on an object already in the shared pool, to verify that the permissions on the underlying object have not changed. | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.pga_memory** | PGA size at the end of the interval | By | Gauge(Int) | <ul> </ul> |
+| **oracledb.session.physical_reads** | Number of physical reads | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.soft_parses** | Number of soft parses | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.user_commits** | Number of user commits. When a user commits a transaction, the redo generated that reflects the changes made to database blocks must be written to disk. Commits often represent the closest thing to a user transaction rate. | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.session.user_rollbacks** | Number of times users manually issue the ROLLBACK statement or an error occurs during a user's transactions | 1 | Sum(Int) | <ul> </ul> |
+| **oracledb.system.resource_limits** | Designates the current utilization and max, initial and current value of a resource limit across processes, sessions, locks, and more. | 1 | Gauge(Int) | <ul> <li>db.oracle.resource.name</li> <li>db.oracle.resource.type</li> </ul> |
+| **oracledb.system.session_count** | Count of sessions. | 1 | Gauge(Int) | <ul> <li>db.oracle.session.type</li> <li>db.oracle.session.status</li> </ul> |
+| **oracledb.tablespace.max_size** | Maximum size of tablespace in bytes. | By | Gauge(Int) | <ul> <li>db.oracle.tablespace.name</li> </ul> |
+| **oracledb.tablespace.size** | Size of tablespace in bytes. | By | Gauge(Int) | <ul> <li>db.oracle.tablespace.name</li> </ul> |
+
+**Highlighted metrics** are emitted by default. Other metrics are optional and not emitted by default.
+Any metric can be enabled or disabled with the following scraper configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: <true|false>
+```
+
+## Resource attributes
+
+| Name | Description | Type |
+| ---- | ----------- | ---- |
+| db.oracle.instance.name | The name of the instance that data is coming from. | Str |
+
+## Metric attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| db.oracle.resource.name | Resource name. Example values are 'processes', 'sessions', 'enqueue_locks', 'enqueue_resources'. |  |
+| db.oracle.resource.type | Resource type, one of 'current_utilization', 'max', 'initial', 'current'. |  |
```

Comment:
I see this attribute is only applied on `oracledb.system.resource_limits` metrics. So it's a but confusing why `current_utilization` and `current` are part of limits metrics? 

Also `oracledb.system.resource_limits` metric itself doesn't seem right. I'd say the metrics should be split and each should be called similar to the actual current value, e.g. `oracledb.enqueue_deadlocks.limit` for `oracledb.enqueue_deadlocks.count`.

Reply:
they are all split now.